docs/component_conventions.md

```diff
@@ -201,3 +201,73 @@ Code can be called and referred to in the template as follows:
 
 <%= card_helper.heading_tag %>
 ```
+
+## Passing HTML to a component
+To avoid the risk of Cross Site Scripting (XSS) attack, we only mark HTML as safe  only sanitise HTML when the component is used, rather than doing so on the application's behalf.
+
+This means avoiding use of 'raw' or 'html_safe' within a component's view.
+
+There are a few methods to achieve this:
+
+### Single slots for nested children
```

Comment:
Maybe instead of slots for children we can talk about parameters with HTML? Would that be clearer?

Reply:
I'm using slots as it's the word used in Web Components, but I think your suggestion is much clearer.

Reply:
This is the first time I've heard of Web Components :)